sdk/communication/communication-administration/review/communication-administration.api.md

```diff
@@ -235,28 +264,26 @@ export interface PageableOptions extends OperationOptions {
 export class PhoneNumberAdministrationClient {
     constructor(connectionString: string, options?: PhoneNumberAdministrationClientOptions);
     constructor(url: string, credential: KeyCredential, options?: PhoneNumberAdministrationClientOptions);
-    cancelSearch(searchId: string, options?: OperationOptions): Promise<RestResponse>;
-    configurePhoneNumber(request: ConfigurePhoneNumberRequest, options?: ConfigurePhoneNumberOptions): Promise<RestResponse>;
-    createSearch(request: CreateSearchRequest, options?: CreateSearchOptions): Promise<PhoneNumberAdministrationCreateSearchResponse>;
-    getAreaCodes(request: GetAreaCodesRequest, options?: GetAreaCodesOptions): Promise<PhoneNumberAdministrationGetAllAreaCodesResponse>;
-    getCapabilitiesUpdate(capabilitiesUpdateId: string, options?: OperationOptions): Promise<PhoneNumberAdministrationGetCapabilitiesUpdateResponse>;
-    getPhoneNumberConfiguration(phoneNumber: string, options?: OperationOptions): Promise<PhoneNumberAdministrationGetNumberConfigurationResponse>;
-    getPhonePlanLocationOptions(request: GetPhonePlanLocationOptionsRequest, options?: GetPhonePlanLocationOptionsOptions): Promise<PhoneNumberAdministrationGetPhonePlanLocationOptionsResponse>;
-    getRelease(releaseId: string, options?: OperationOptions): Promise<PhoneNumberAdministrationGetReleaseByIdResponse>;
-    getSearch(searchId: string, options?: OperationOptions): Promise<PhoneNumberAdministrationGetSearchByIdResponse>;
+    cancelSearch(searchId: string, options?: SearchOptions): Promise<VoidResponse>;
```

Comment:
Should this be `CancelSearchOptions`?

Reply:
Same for `purchaseSearch` and `refreshSearch`. But imo `SearchOptions` for all of them isn't too bad, so not a strong opinion here.